README.md

```diff
@@ -238,3 +238,18 @@ pattern.
 [pytest]: https://docs.pytest.org/en/latest/
 [mozilla]: https://github.com/mozilla/addons-server
 [selenium-training]: https://qualitas-server.herokuapp.com/wiki/Introduction_to_Pytest_and_Selenium
+
+### HOW TO CREATE TEST DATA FOR REDIRECTS
+
+1. go to ce-scripts repo
+2. cd python
```

Comment:
Perhaps we should add a note to create a `venv` and run `pip install -r requirements.txt` here prior to running `gen_book_uris.py`?

Reply:
OK